db/migrate/20160517184709_add_direct_otp_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDirectOtpToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :direct_otp, :string
+    add_column :users, :direct_otp_sent_at, :datetime
+  end
+end
```

Comment:
I don't think thats needed.   There are only a million possible 6 digit OTPs so collisions are not inconceivable.